content/blog/wingingit/winging-it-03.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Intuitive Web Design with OOUX
+episode: 3
+tags:
+  - OOUX
+  - Design
+author:
+  - sondra
+  - james
+  - david
+date: 2023-10-24
+length: 32 mins
+image:
+  src: winging-it/winging-it-3.jpg
+media:
+  - iframe: https://www.youtube.com/embed/wk5tFSqYVzs
+summary: |
+  During our conversation we look at 3 examples of unintuitive web design, and
+  learn how to create websites and apps that are more intuitive using
+  object-oriented user experience (OOUX) design strategies.
+---
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+
+- What is OOUX
+- Practical examples for discovering objects
+- Types of objects including broken, shapeshifter, and isolated objects
+- Object maps
+- OOUX Certification
+
+## Links:
+
+- [Denver Zoo](https://denverzoo.org/)
+- [OOUX Certification](http://partners.ooux.com/348125/17189)
+- [Slides](https://xd.adobe.com/view/59a38984-ff)
```

Comment:
```suggestion
- [Slides](https://xd.adobe.com/view/59a38984-ff8c-4ae6-b09f-2cbf31f235fc-0cf0/)
```
The link is cut off, here's the one from Youtube

Reply:
@jamesnw fixed in https://github.com/oddbird/oddleventy/pull/436/commits/169a5d6c1de86d698c2bfa7e00354c40863fb205. Thanks for catching that. I thought it was weird that it was so short. I should have known to test it.